src/lib/feature-flags/flags.ts

```diff
@@ -0,0 +1,87 @@
+import window from 'global/window';
+
+export type FeatureFlagKey =
+  | 'channelsApp'
+  | 'enableDevPanel'
+  | 'resetPasswordPage'
+  | 'enableRewards'
+  | 'enableMatrix'
+  | 'allowEmailRegistration'
+  | 'verboseLogging'
+  | 'allowEditPrimaryZID'
+  | 'enableReadReceiptPreferences'
+  | 'enableUserSettings'
+  | 'enableTimerLogs'
+  | 'enableChannels'
+  | 'enableCollapseableMenu'
+  | 'enableMeows'
+  | 'enableTokenGatedChat'
+  | 'enableNotificationsApp'
+  | 'enableNotificationsReadStatus'
+  | 'enableLoadMore'
+  | 'enableComments'
+  | 'enableFeedApp'
+  | 'enableLinkedAccounts'
+  | 'enableZeroWalletSigning'
+  | 'enableFeedChat';
```

Comment:
I'd be happy to clean up any of these if we no longer need them

Reply:
Yeah we could definitely do with cleaning these up, most are no longer required